lib/internal/source_map/source_map_cache.js

```diff
@@ -77,7 +78,22 @@ function setSourceMapsEnabled(val) {
   sourceMapsEnabled = val;
 }
 
-function maybeCacheSourceMap(filename, content, cjsModuleInstance, isGeneratedSource) {
+function extractSourceURLMagicComment(content) {
```

Comment:
Is this additional work from your adding support for source maps in evaled code?  `sourceURL=` specifically applies to this right?

Reply:
No, it's not needed for mapping the stack lines. This is needed to caching the source URL for [GetSourceMapErrorSource](https://github.com/nodejs/node/pull/43875/files#diff-090d65062da5652b79f8a641489e4bd33b37faf28b8e91cc2ac31ab43084a3d2R60) as explained in the inline comments.

Reply:
Gotcha, so v8 itself consumes the magic URL and `message->GetScriptResourceName();` will return the URL in the comment 👍